docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,100 @@
+# Rocket.Chat help: developing stronger questions
+Last updated: **April 16, 2023**
```

Comment:
```suggestion
Last updated: **April 16, 2024**
```

Reply:
I think we can remove these now that we have the automatic date!